eth/tracers/tracers_test.go

```diff
@@ -298,3 +344,103 @@ func jsonEqual(x, y interface{}) bool {
 	}
 	return reflect.DeepEqual(xTrace, yTrace)
 }
+
+// Iterates over all the input-output datasets in the tracer parity test harness and
+// runs the JavaScript tracers against them.
+func TestCallTracerParity(t *testing.T) {
+	files, err := ioutil.ReadDir("testdata")
+	if err != nil {
+		t.Fatalf("failed to retrieve tracer test suite: %v", err)
+	}
+	for _, file := range files {
+		if !strings.HasPrefix(file.Name(), "parity_call_tracer_") {
+			continue
+		}
+		file := file // capture range variable
+		t.Run(camel(strings.TrimSuffix(strings.TrimPrefix(file.Name(), "parity_call_tracer_"), ".json")), func(t *testing.T) {
```

Comment:
Instead of `strings.TrimPrefix` it might be more idiomatic to use `filepath.Base`.